integration-tests/smoke/ccip/ccip_usdc_test.go

```diff
@@ -173,28 +178,41 @@ func TestUSDCTokenTransfer(t *testing.T) {
 		},
 	}
 
+	for i, tt := range tcs {
+		tcs[i].initialBalances = make(map[common.Address]*big.Int)
```

Comment:
I think this doesn't seem right. Every test can pick any token / any chain to transfer. Therefore any sequential test relies on the outcome of the previous one. For instance, if test1 sends a token from A -> B, and test2 does a similar work, you will get the wrong initial balance for test2, because in the meantime that balance will be changed by test1 execution.

If it doesn't fails then probably it's mainly because we pick different combinations for every test case. However, adding new test case will probably break it, no?

Reply:
Every test is using a different receiver right? will that not make it separate from each other